guides/source/routing.md

```diff
@@ -1232,41 +1296,28 @@ Rails now creates routes to the `CategoriesController`.
 | PATCH/PUT | /kategorien/:id            | categories#update  | category_path(:id)      |
 | DELETE    | /kategorien/:id            | categories#destroy | category_path(:id)      |
 
-### Overriding the Singular Form
-
-If you want to override the singular form of a resource, you should add additional rules to the inflector via [`inflections`][]:
-
-```ruby
-ActiveSupport::Inflector.inflections do |inflect|
-  inflect.irregular 'tooth', 'teeth'
-end
-```
-
-[`inflections`]: https://api.rubyonrails.org/classes/ActiveSupport/Inflector.html#method-i-inflections
-
 ### Using `:as` in Nested Resources
```

Comment:
I wonder if this should be part of `Overriding the Named Route Helpers`

Reply:
I think it works here too as the nested example is a bit more complicated and having it in two places reinforces the information. Resolving for now, but let me know if you feel strongly, open to restructuring as well.